.evergreen-functions.yml

```diff
@@ -258,6 +265,14 @@ functions:
           - ${workdir}/venv/bin
         working_dir: src/github.com/mongodb/mongodb-kubernetes
         binary: scripts/evergreen/check_precommit.sh
+    - command: subprocess.exec
```

Comment:
I would like this to be part of check_precommit/linting script

Reply:
Since we didn't discuss this in last discuss meeting @lsierant. Is it ok if we handle this comment later after the discussion, so that we can merge this PR? Right now I have added a way to run this script locally.